ui/src/app/applications/components/applications-list/applications-tiles.tsx

```diff
@@ -139,7 +139,13 @@ export const ApplicationTiles = ({applications, syncApplication, refreshApplicat
                                                         </div>
                                                         <div className={app.status.summary.externalURLs?.length > 0 ? 'columns small-2' : 'columns small-1'}>
                                                             <div className='applications-list__external-link'>
-                                                                <ApplicationURLs urls={app.status.summary.externalURLs} />
+                                                                <a
+                                                                    href={linkInfo.url}
+                                                                    target={linkInfo.isExternal ? '_blank' : undefined}
+                                                                    rel={linkInfo.isExternal ? 'noopener noreferrer' : undefined}
+                                                                    title={`Link: ${linkInfo.url}\nmanaged-by-url: ${getManagedByURL(app) || 'none'}`}>
+                                                                    <i className='fa fa-external-link-alt' />
+                                                                </a>
```

Comment:
Look for examples elsewhere where we run ctx.navigation.goto for handling onClick

This allows you to click on the forward and back buttons in the browser and it will navigate properly.